searx/searx
# Add Springer Nature engine

## What does this PR do?

Add a search engine for Springer Nature: https://www.springernature.com/ .

## Why is this change important?

Springer Nature is a major player in the scientific community, its publications deserve to be searched.

## How to test this PR locally?

First get your API key following this page:
https://dev.springernature.com/signup

In searx/engines/springer.py
at line 21, add this API key.
I left my own key, commented out on line 20.  Feel free to use it, if needed.

In searx, try a search using some scientific terms, links to Springer Nature should be listed.
Here is an example of a screenshot from such a search:
![image](https://user-images.githubusercontent.com/2790849/111498449-40d19380-8742-11eb-9091-f4167cc377cf.png)



## Related issues

None
